### PR TITLE
Add tray item to switch "Bypass CN Mainland" on/off

### PR DESCRIPTION
**Pull request by kqwyf**
image

Tested on Manjaro KDE at 2021-03-05.

Solves #627 .

### CODE REVIEW

File: translations/zh_CN.ts [hunk 1]
```diff
@@ -956,6 +960,14 @@ This entry is ignored by V2Ray core when using DoH servers.</source>
...
         <translation>禁用系统代理</translation>
     </message>
+    <message>
+        <source>Enable Bypass CN Mainland</source>
+        <translation>启用绕过中国大陆</translation>
+    </message>
+    <message>
+        <source>Disable Bypass CN Mainland</source>
+        <translation>禁用绕过中国大陆</translation>
+    </message>
```
Comment (flylai) [inline translations/zh_CN.ts:970]: You need to commit the changes to en_US.ts instead of zh_CN.ts

**Comment by U-v-U** [comment]
do not use the `master` branch, use `dev` instead.

File: translations/zh_CN.ts (hunk 1 above)
Comment (kqwyf) [reply translations/zh_CN.ts:970]: I'm sorry but I don't understand how the translation mechanism works. I'll commit a new PR without translation to the dev branch. Could you kindly give me some directions on making translation?

**Comment by kqwyf** [comment]
Moved to #1335 .

File: translations/zh_CN.ts (hunk 1 above)
Comment (flylai) [reply translations/zh_CN.ts:970]: I have no idea, it may be that my system language is English and it automatically generates English.

**Pull request closed without merging**